test/integration_test.go

```diff
@@ -5169,15 +5170,45 @@ func TestStartsWith(t *testing.T) {
 
 	require.NoError(t, json.Unmarshal([]byte(stdout.String()), &outputs))
 
-	validateBoolOutput(t, outputs, "startswith1", true)
-	validateBoolOutput(t, outputs, "startswith2", false)
-	validateBoolOutput(t, outputs, "startswith3", true)
-	validateBoolOutput(t, outputs, "startswith4", false)
-	validateBoolOutput(t, outputs, "startswith5", true)
-	validateBoolOutput(t, outputs, "startswith6", false)
-	validateBoolOutput(t, outputs, "startswith7", true)
-	validateBoolOutput(t, outputs, "startswith8", false)
-	validateBoolOutput(t, outputs, "startswith9", false)
+	validateOutput(t, outputs, "startswith1", true)
+	validateOutput(t, outputs, "startswith2", false)
+	validateOutput(t, outputs, "startswith3", true)
+	validateOutput(t, outputs, "startswith4", false)
+	validateOutput(t, outputs, "startswith5", true)
+	validateOutput(t, outputs, "startswith6", false)
+	validateOutput(t, outputs, "startswith7", true)
+	validateOutput(t, outputs, "startswith8", false)
+	validateOutput(t, outputs, "startswith9", false)
+}
+
+func TestTimeCmp(t *testing.T) {
```

Comment:
I was wondering if we need to have a test for invalid date

Reply:
Fair remark! Added invalid date integration test